packages/cascara/src/ui/Table/TableHeader.js

```diff
@@ -0,0 +1,83 @@
+import React, { useContext } from 'react';
+
+import Button from '../Button';
+import DownloadButton from '../../modules/DownloadButton';
+
+import TableContext from './context';
+import SelectionToggle from './atoms/SelectionToggle';
+import ActionBar from './ActionBar';
+
+const TableHeader = ({ footer }) => {
```

Comment:
I think we should actually make TableHeader and TableFooter separate components entirely. The reason for this is eventually the styles are going to be tied to these two areas directly with modules. And the logic inside them may be different. 

- [ ] Maybe set up a separate TableFooter component?

Reply:
sure thing, we will have those as separate components.